packages/release-mastermind/src/action.ts

```diff
@@ -12,314 +1,535 @@
+					config[curContext.type]![action] = config.sharedConfig[action]
+				}
+			}
+			core.endGroup()
+			this.applyContext(configs, config, curContext)
+		})
+	}
+
+	/**
+	 * Get the configuration
+	 * @author IvanFon, TGTGamer, jbinda
+	 * @since 1.0.0
+	 */
+	async processConfig(): Promise<Runners> {
+		if (!this.configJSON?.runners[0]) {
+			if (!fs.existsSync(this.configPath)) {
+				throw new Error(`config not found at "${this.configPath}"`)
+			}
+			const pathConfig = await JSON.parse(
+				fs.readFileSync(this.configPath).toString()
+			)
+			if (!pathConfig.releaseMastermind) return pathConfig
+			else return pathConfig.releaseMastermind
+		} else {
+			return this.configJSON
+		}
+	}
+
+	/**
+	 * Handle the context
+	 * @author IvanFon, TGTGamer, jbinda
+	 * @since 1.0.0
+	 */
+	async processContext(config: Config) {
+		let curContext: CurContext
+
+		if (context.payload.pull_request) {
+			/**
+			 * Pull Request Context
+			 * @author IvanFon, TGTGamer, jbinda
+			 * @since 1.0.0
+			 */
+			const ctx = await PullRequests.parse(this.util, config, context).catch(
+				(err) => {
+					throw log(
+						LoggingLevels.error,
+						`Error thrown while parsing PR context: `,
+						err
+					)
+				}
+			)
+			if (!ctx) {
+				throw new LoggingDataClass(
+					LoggingLevels.error,
+					"Pull Request not found on context"
+				)
+			}
+			log(LoggingLevels.debug, `PR context: ${JSON.stringify(ctx)}`)
+			curContext = {
+				type: "pr",
+				context: ctx
+			}
+		} else if (context.payload.issue) {
+			/**
+			 * Issue Context
+			 * @author IvanFon, TGTGamer, jbinda
+			 * @since 1.0.0
+			 */
+			const ctx = await Issues.parse(this.util, config, context).catch(
+				(err) => {
+					throw log(
+						LoggingLevels.error,
+						`Error thrown while parsing issue context: ` + err
+					)
+				}
+			)
+			if (!ctx) {
+				throw new Error("Issue not found on context")
+			}
+			log(LoggingLevels.debug, `issue context: ${JSON.stringify(ctx)}`)
+
+			curContext = {
+				type: "issue",
+				context: ctx
+			}
+		} else if (context.payload.project_card) {
+			/**
+			 * Project Context
+			 * @author TGTGamer
+			 * @since 1.0.0
+			 */
+			const ctx = await Project.parse(this.util, config, context).catch(
+				(err) => {
+					log(
+						LoggingLevels.error,
+						`Error thrown while parsing Project context: `,
+						err
+					)
+					return err
+				}
+			)
+			if (!ctx) {
+				throw new Error("Project Card not found on context")
+			}
+			log(LoggingLevels.debug, `Project Card context: ${JSON.stringify(ctx)}`)
+
+			curContext = {
+				type: "project",
+				context: ctx
+			}
+		} else if (context.payload.schedule) {
+			/**
+			 * Project Schedule Context
+			 * @author TGTGamer
+			 * @since 1.0.0
+			 */
+			const ctx = await Schedule.parse(context).catch((err) => {
+				log(
+					LoggingLevels.error,
+					`Error thrown while parsing Schedule context: `,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
            "Error thrown while parsing Schedule context: ",
```